js/core/include-config.js

```diff
@@ -0,0 +1,20 @@
+// Module core/include-config
+// Inject's the document's configuration into the head as JSON.
+
+define(
+  [],
+  function() {
+    return {
+      run: function(conf, doc, cb, msg) {
+        msg.pub("start", "core/include-config");
+        var script = doc.createElement("script");
+        script.id = "respecConfig";
```

Comment:
Please choose a different id for this element as it'll end up in the global scope thanks to the ridiculous legacy feature that DOM ids are globals. I have plenty of script which test for `window.respecConfig` to decide whether or not client-side rendering is going on. That'll break them.